static/socket/logrotate

```diff
@@ -0,0 +1,8 @@
+/var/www/html/pathfinder/history/map/*.log {
+	daily
+	missingok
+	rotate 14
+	compress
+	notifempty
+	endscript
```

Comment:
Yep, thanks. I copied from a config that had a postscript block and missed deleting this line.

Reply:
@samoneilll Youre also missing the volume mount to pathfinder container so it can read the logs

Reply:
Does it need to read the logs? I'm not actually aware of that functionality, but if you can tell me where it should be mounted I can try it.

Reply:
yes, pathfinder container needs to read the logs. It should be mounted to the same directory, but in pathfinder container.

Logrotate should also probably be in the pathfinder container, and running through supervisord as the socket container has no way to run logrotate

Reply:
the composer image the socket container is built from already uses logrotate for acpid, but you're right that it makes more sense in the PF container, especially since we have more control over that image. I'll adjust.

Reply:
@JorisBolsens can you check when you have some time? 

I've mounted the logs to the PF service, and added logrotate and a config in the Dockerfile. Works fine on my development instance and PF can read the logs.

Reply:
@samoneilll I am not 100% sure, but I think you need to add logrotate to cron aswell:
```
02 4 * * * /usr/sbin/logrotate /etc/logrotate.d/
```

Reply:
logrotate adds itself to /etc/period/daily/ when it installs, which should be executed by crond.

it looks like it should work okay: 

`pathfinder:~#     run-parts --test /etc/periodic/daily
/etc/periodic/daily/logrotate`

Reply:
awesome

Reply:
I'm not 100% either because I don't fully understand how the scripts /etc/periodic are picked up by crond, but according to the alpine documentation for it there doens't appear to be any additional configs required. I'll run it for a few days and check that it actually executes.